### PR TITLE
add lock and unlock commands

### PR DESCRIPTION
**Pull request by vdbg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ Device groups represent collection of Hubitat devices that can be accessed by us
...
 * `NONE`: cannot use any commands. Useful to disable a user group.
 * `DEVICE`: can use device commands e.g., `/list`, `/on`, `/off`, `/open`, `/close`, `/dim`, `/status`, `/commands`.
-* `HSM`: can use device commands and also [Hubitat Safety Monitor](https://docs.hubitat.com/index.php?title=Hubitat%C2%AE_Safety_Monitor_Interface) commands e.g, `/mode`, `/arm`, `/events`.
-* `ADMIN`: can use device, hsm and also admin commands e.g., `/users`, `/groups`, `/info`, `refresh`. In addition some commands have more detailed output (e.g., `/list`, `/status`).
+* `SECURITY`: can use device commands, the `/arm` command for [Hubitat Safety Monitor](https://docs.hubitat.com/index.php?title=Hubitat%C2%AE_Safety_Monitor_Interface), the `/mode` command to view and change the mode, the `/events` command to see a device's history, and act on locks with `/lock`, `/unlock`.
+* `ADMIN`: can use the device, security and also admin commands e.g., `/users`, `/groups`, `/info`, `refresh`. In addition some commands have more detailed output (e.g., `/list`, `/status`).
```
Comment (Nickolaim) [inline README.md:40]: I cannot make the sense out of `can use the device, security and`

File: main.py [hunk 2]
```diff
@@ -91,8 +91,8 @@ def is_allowed_device(device: Device) -> bool:
             supported_commands = set()
 
             for command in commands:
-                bot_command = self.hubitat.he_to_bot_commands.get(command, None)
-                if bot_command:
+                if command in self.hubitat.he_to_bot_commands:
```
Comment (Nickolaim) [inline main.py:94]: Something is wrong in this and the next lines.

The next line uses `[]` and I assume that `he_to_bot_commands` is a dict.  This line does `foo in dict` that is wrong, one usually searches in keys or values

Comment (vdbg) [reply main.py:94]: Am I missing something or misunderstanding your comment?
```
>>> test = { "foo": "bar" }
>>> "foo" in test
True
>>> "bar" in test
False
>>> "foobar" in test
False
>>> test["foo"]
'bar'
```

Comment (Nickolaim) [reply main.py:94]: Well, this is news for me.  No, you code makes sense.  Python behavior used to be - do `in for .keys()` Looks like this is the default now for in operator.  

Your code is perfect!

File: README.md (hunk 1 above)
Comment (vdbg) [reply README.md]: should make more sense now

**Pull request merged**